src/cli/self_update.rs

```diff
@@ -390,13 +397,12 @@ fn check_existence_of_rustc_or_cargo_in_path(no_prompt: bool) -> Result<()> {
         err!("Otherwise you may have confusion unless you are careful with your PATH");
         err!("If you are sure that you want both rustup and your already installed Rust");
         err!("then please restart the installation and pass `-y' to bypass this check.");
```

Comment:
This is the message which needs tweaking now that you can answer `y` rather than **having** to pass `-y`

Reply:
Should I still have the prompt, or force the (possibly overly broad) command-line argument?

Reply:
I still want the prompt, the point is that the message needs tweaking *because* we want the prompt :D

Reply:
Yep, makes sense, just double-checking.